libpod/reset.go

```diff
@@ -87,12 +88,28 @@ func (r *Runtime) Reset(ctx context.Context) error {
 		}
 		prevError = err
 	}
-	if err := os.RemoveAll(r.config.TmpDir); err != nil {
-		if prevError != nil {
-			logrus.Error(prevError)
+
+	runtimeDir, err := util.GetRuntimeDir()
+	if err != nil {
+		return err
+	}
+	tempDirs := []string{r.config.TmpDir}
+	if r.config.TmpDir == runtimeDir {
+		tempDirs = []string{
+			filepath.Join(r.config.TmpDir, r.store.GraphDriverName()+"-containers"),
```

Comment:
@rhatdan should this also remove `filepath.Join(r.config.TmpDir, r.store.GraphDriverName())` ?

Reply:
Yes if it exists.

Reply:
should  "containers" under TmpDir in the path that we need to remove?
`/TmpDir/containers/overlay` instead of `/TmpDir/overlay`

or we just need to remove `/TmpDir/containers` instead of hardcode these path all

Reply:
Just remove containers.